content/docs/learning-guide/tutorials/environments/generate-terrain.md

```diff
@@ -0,0 +1,303 @@
+
+* 16-bit color depths are recommended. 8-bit images may not provide enough gradient steps to represent elevation, and 32-bit images might be unnecessarily large. 16-bit images can provide up to 65,536 elevation steps.
+
+* Ensure the image is square with a power of two resolution (for example, 256 x 256, 512 x 512, 1024 x 1024, and so on). For the best result, each pixel in the heightmap should represent one world meter. A 1024 x 1024 heightmap, for example, represents just over one square kilometer of terrain.
+
+* Use `.png` or `.tif` formats for 16-bit heightmap images.
+
+* Use `.tif` format for 32-bit heightmap images.
+
+* Postfix the heightmap file name with `_gsi` so that the heightmap is automatically processed by Asset Processor as a GSI (`LevelOneTerrainHeight_gsi.png`, for example). If you don't use the `_gsi` postfix, you can use [Texture Settings](/docs/user-guide/assets/texture-settings/) to configure the heightmap source asset to be processed as a GSI.
+
+## Enable the terrain system
+
+First, you must enable the terrain system by adding two level components to the level prefab.
+
+1. In **O3DE Editor**, create a new level or open an existing level.
+
+    {{< tip >}}
+If your level contains the **Atom Default Environment**, delete or hide the Grid, Ground, and Shader Ball entities so that the terrain you build is not visually obstructed.
+    {{< /tip >}}
+
+2. In **Entity Outliner**, select the Level.prefab.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/enable-terrain-select.png" alt="Select the level.prefab." >}}
+
+3. In **Entity Inspector**, choose **Add Component** and add both the **Terrain World** and **Terrain World Renderer** level components to the level.prefab.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/enable-terrain-components.png" width="600" alt="Add terrain level components." >}}
+
+There are many properties available in the terrain level components that configure the terrain settings for the level. Later in the tutorial, you might need to make some adjustments to these properties.
+
+{{< important >}}
+The default mapping of a heightmap image is one pixel to one world unit in O3DE. This means that in order to see the full resolution of the heightmap as a terrain, the terrain must be as large (in world units) as the heightmap (in pixels). Even for small heightmaps, you might need to make the following temporary adjustments so that you can view the entire terrain at full resolution while you build it:
+
+1. In the **Terrain World Renderer** level component, set both the **Mesh render distance** and the **First LOD distance** properties to a value that is greater than the resolution of the heightmap.
```

Comment:
This seemed like too much info for the first step. I broke it down into multiple steps/substeps and tried to use fewer words. Does this work:

1. In the Terrain World Renderer level component, set the **Mesh render distance** to a value that is greater than the resolution of the heightmap, so that the entire terrain will be visible.

2. If the heightmap is larger than 256 x 256 do the following:

   1. Set the **First LOD distance** property to `256`. This sets the terrain's highest LOD display to a distance of 256 meters from the camera.

   2. Set the **CLOD Distance** property to `32`. This sets the blend distance between LODs. Larger values make the LOD transitions smoother, but the value should be less than 1/4 the **First LOD distance** value to ensure that there is enough distance blend each LOD.

Reply:
Looks good, except "enough distance blend" -> "enough distance to blend".

Reply:
fixed